### DISCUSSION TITLE
Using PatchCreation

### DISCUSSION THREAD

**Discussion by yuvnith** [wixtoolset/issues#8149] [Questions] [answered]
We are looking for ways to use WiX v4 to build an MSP file, and in the process, we discovered that we could use the Patch Creation Process. 
When using PatchCreation and "Patch" as the OutputType in.wixproj, we receive a compiler error that states 
`No transforms were provided to attach to the patch.`

However, if we set "PatchCreation" as the OutputType, we receive a compiler error saying
`Unable to find a backend to process output type`

After exploring through the WiX 4* source code, found that OutputType `PatchCreation `was commented in `WindowsInstallerBackendFactory.cs` which uses `PatchCreationBackend`

Could someone be kind enough to clarify to me on how to utilize patchcreation correctly and as expected?

**Comment by robmen** [comment, marked as answer]
Patch creation has never been supported using a .wixproj. It would have been _really hard_ in WiX v3. In WiX v4+, it's probably not nearly as hard, but no one has done the additional work to enable it. Today you have to use the command-line to build patches.

**Reply by yuvnith** [reply]
Thank you for the response @robmen
